docs/getting-started.md

```diff
@@ -282,3 +282,24 @@ If you are not able to send custom headers in your handshake request, send it as
 query parameter `authorization`. Then your path should look like this
 `/channels/<channel_id>/messages?authorization=<client_auth_key>`.
 
+### CoAP
+
+To publish message over channel, client should send following confirmable message:
+
+```
+coap post coap://localhost/channels/<channel_id>/messages?key=<client_token>
+```
+
+Client token must be present in `Uri-Query` option. Otherwise, message will be considered unauthorized.
+
+To subscribe to channel, client should send following confirmable message:
+
+```
+coap get coap://localhost/channels/<channel_id>/messages?key=<client_token> -o
+```
+
+where `-o` in indicates that value of `Observe` option is 0.
+To unsubscribe from channel, client should send same message with value of `Observe` option set to 1
+or ignore confirmable message.
+
+Useful tools for sending CoAP messages can be found [here](https://github.com/mcollina/coap-cli) and [here](https://github.com/mkovatsc/Copper).
```

Comment:
Put this at the beginning and name the links according to the tool names, instead of as "here".